app/components/ObservationsInspector.tsx

```diff
@@ -1,166 +1,77 @@
-    <Tooltip content={t('explore.inspect.overrideTooltip')} position={Position.RIGHT}>
-      <div className={styles.predictionOverride}>
-        <h4 className={styles.predictionLabel}>{t('explore.inspect.override')}</h4>
-        <CreatableSelect
-          name={predictionOverride}
-          value={predictionOverride}
-          onChange={handlePredictionOverride}
-          isClearable
-          options={taxonOptions}
-          menuShouldScrollIntoView={false}
-        />
-      </div>
-    </Tooltip>
-  );
-  const photoDetails = (
-    <div className={styles.photoDetails}>
-      <p>
-        <strong>
-          {t('explore.inspect.camera')}
-          :&nbsp;
-        </strong>
-        {observation.camera}
-      </p>
-      <p>
-        <strong>
-          {t('explore.inspect.file')}
-          :&nbsp;
-        </strong>
-        {path.basename(observation.location)}
-      </p>
-    </div>
-  );
-
-  return (
-    <div className={styles.observation}>
-      <Card className={styles.card} elevation={Elevation.TWO} key={observation.location}>
-        <h3 className={styles.heading}>
-          {t('explore.inspect.photoHeader', {
-            species: formatAnimalClassName(
-              predictionOverride ? predictionOverride.value : observation.pred_1
-            ),
-            date: observation.date
-          })}
-        </h3>
-        <div className={styles.body}>
-          <div className={styles.photo}>
-            <img
-              className={styles.img}
-              src={observation.location}
-              width={360}
-              alt={observation.pred_1}
-            />
-          </div>
-          <div className={styles.data}>
-            {predictionsTable}
-            {predictionOverrideWidget}
-            {photoDetails}
-          </div>
-        </div>
-      </Card>
-    </div>
-  );
-}
-
 type ObservationsInspectorProps = {
+  isVisible: boolean;
   observations: Observation[];
   onClose: () => void;
   predictionOverrides: PredictionOverridesMap;
   onPredictionOverride: PredictionOverrideHandler;
+  onPhotoClick?: (cardIndex: number) => void;
 };
 
 export default function ObservationsInspector(props: ObservationsInspectorProps) {
-  const { observations, onClose, predictionOverrides, onPredictionOverride } = props;
+  const { isVisible, observations, onClose, predictionOverrides, onPredictionOverride } = props;
+  if (!isVisible) return null;
   const { t } = useTranslation();
-  if (observations.length === 0) {
-    return null;
-  }
+  const [maximizedCardIndex, setMaximizedCardIndex] = useState<number>(-1);
+  const [isMaximizedMode, setMaximizedMode] = useState<boolean>(false);
+  const toggleMaximizedMode = (cardIndex: number) => {
+    setMaximizedCardIndex(cardIndex);
+    setMaximizedMode(cardIndex >= 0);
+  };
+  const backButtonText = t(`explore.${isMaximizedMode ? 'backToObservations' : 'backToMap'}`);
+  const onBackButtonClick = isMaximizedMode ? () => toggleMaximizedMode(-1) : onClose;
```

Comment:
I find this slightly easier to read, WDYT? :wink:
```suggestion
  const onBackButtonClick = () => isMaximizedMode ? setMaximizedCardIndex(null) : onClose();
```

Reply:
agree, fixed